docs/content/management-decision-logs.md

```diff
@@ -224,9 +228,11 @@ the following rule format can be used.
 ```ruby
 package system.log
 
+import rego.v1
+
 # always upsert, no conditions in rule body
-mask[{"op": "upsert", "path": "/input/password", "value": x}] {
-  x := "**REDACTED**"
+mask contains {"op": "upsert", "path": "/input/password", "value": x} if {
```

Comment:
While we're at it, I think we could just change this to be:
```rego
mask contains {"op": "upsert", "path": "/input/password", "value": "**REDACTED**"}
```
And do the same at line 218. Not much point in that extra variable.

Reply:
Updated.

Reply:
Thanks!